pkg/resource/events-logger-config/grafana-agent-config.go

```diff
@@ -29,27 +25,15 @@ var (
 
 func init() {
 	grafanaAgentTemplate = template.Must(template.New("events-logger.grafanaagent").Funcs(sprig.FuncMap()).Parse(grafanaAgent))
-	grafanaAgentConfigTemplate = template.Must(template.New("events-logger-config.grafanaagent.yaml").Funcs(sprig.FuncMap()).Parse(grafanaAgentConfig))
-}
-
-// configMeta returns metadata for the grafana-agent-config
-func configMeta(lc loggedcluster.Interface) metav1.ObjectMeta {
-	metadata := metav1.ObjectMeta{
-		Name:      getGrafanaAgentConfigName(lc),
-		Namespace: lc.GetAppsNamespace(),
-		Labels:    map[string]string{},
-	}
-
-	common.AddCommonLabels(metadata.Labels)
-	return metadata
+	grafanaAgentConfigTemplate = template.Must(template.New("events-logger.grafanaagent.yaml").Funcs(sprig.FuncMap()).Parse(grafanaAgentConfig))
 }
 
-// generateGrafanaAgentConfig returns a configmap for
+// GenerateGrafanaAgentConfig returns a configmap for
 // the grafana-agent extra-config
-func generateGrafanaAgentConfig(lc loggedcluster.Interface, defaultWorkloadClusterNamespaces []string) (string, error) {
+func generateGrafanaAgentConfig(lc loggedcluster.Interface, defaultNamespaces []string) (string, error) {
```

Comment:
Could you rename it to defaultWorkloadClusterNamespaces? This makes it semantically more correct